### PR TITLE
Manage in app purchases

### PR DESCRIPTION
**Pull request by gregg-platogo**
- [x] Add `bin/console` script 
- [x] List price points for an app
 - [x] Listing In-App Purchases and their details 
 - [x] Manage In-App Purchases

### CODE REVIEW

**Comment by kyledecot** [comment]
Thanks for the contribution! I released these changes as [v0.33.0](https://rubygems.org/gems/app_store_connect/versions/0.33.0) 💯

File: bin/console [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
```
Comment (kyledecot) [inline bin/console:1]: @gregg-platogo  @tmarkanov There is already something similar via `bundle exec rake console` which automatically instantiates a client.

Comment (gregg-platogo) [reply bin/console:1]: Right we missed that! BTW it would be cool if you added CHANGELOG to the releases 😉

Comment (kyledecot) [reply bin/console:1]: Done! https://github.com/kyledecot/app_store_connect/blob/main/CHANGELOG.md

Comment (gregg-platogo) [reply bin/console:1]: Awesome! Thanks!

**Pull request merged**